app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,4 +1,9 @@
 // Specific CSS for your home-page
+body {
```

Comment:
make sure this `body` doesn't affect other web pages.

would be better if you add a `id="banner"` or something and make the background image there instead